catroid/src/main/java/org/catrobat/catroid/ui/fragment/FormulaEditorCategoryListFragment.java

```diff
@@ -369,7 +372,14 @@ public void onStart() {
 		int[] noParametersList = new int[length];
 
 		for (int i = 0; i < length; i++) {
-			noParametersList[i] = R.string.formula_editor_function_no_parameter;
+			//Dirty hack until further insight is gained
+			try {
+				Log.i(TAG, "Trying to get string resource: " + getString(R.string
+						.formula_editor_function_no_parameter));
+				noParametersList[i] = R.string.formula_editor_function_no_parameter;
```

Comment:
This works without any problems?
When the exception is thrown, noParametersList[i] stays 0.
→ `getString(0) == ""`?